README.md

```diff
@@ -108,6 +108,13 @@ Update the host.
 $ xe host-call-plugin host-uuid=<uuid> plugin=updater.py fn=update
 ```
 
+#### `is_installed`:
+Test if one or many packages are installed.
```

Comment:
```suggestion
Return an array of the installed packages given in arguments with an empty entry for not installed ones.
```

Reply:
Not an array. It's JSON Object. ;)
So maybe:
```
Return a JSON object of the installed packages given in arguments with an empty value string for not installed ones.
```

Reply:
Fine by me, I phrased like the method above to keep consistency but your suggestion is ok too.